Gemfile

```diff
@@ -21,13 +21,13 @@ gem 'bcrypt', require: false
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'cancancan'
 # explicit soft-deletes
```

Comment:
this comment can be removed too - it relates to the `discard` gem but I really don't see much value in the comment. Apologies it's unrelated to your PR but the placement of the new gem makes it really apparent!